app/build.gradle

```diff
@@ -35,4 +62,22 @@ dependencies {
 
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+
+    implementation 'com.android.support.constraint:constraint-layout:1.1.0'
+    implementation 'com.google.firebase:firebase-core:16.0.0'
```

Comment:
why different versions for code and authentication?

Reply:
Hi @tushar-acharya, as per https://firebase.google.com/support/release-notes/android both core and auth have different versions. Is this okay?

Reply:
oh ok. let's try with 16.0.1 for core but if it says unavailable, then lets use 16.0.0.

Reply:
Yes I tried with 16.0.1 and the build failed